ipaserver/plugins/cert.py

```diff
@@ -898,7 +898,7 @@ def execute(self, csr, all=False, raw=False, chain=False, **kw):
         store = profile['result']['ipacertprofilestoreissued'][0] == 'TRUE'
         if store and 'certificate' in result:
             cert = result.get('certificate')
-            kwargs = dict(addattr=u'usercertificate={}'.format(cert))
+            kwargs = dict(usercertificate=cert)
```

Comment:
I think that we should use `usercertificate=cert` only when `principal_type in [SERVICE, HOST]`,
and continue using `addattr` for user principals.  Replacing all existing `usercertificate` values doesn't seem right for user principals (even if `user_mod` does not revoke the removed certificates).

Reply:
If the certificate isn't revoked then there is no reason to remove the others. I think a broader discussion is needed on freeipa-devel because this is going to lead to other side-effects, like certs never disappearing from usercertificate on expiration.